canonical/juju-dashboard
# WD-5780 - Add pagination to bottom of table

## Done

- Add back to first page button.
- Add pagination to bottom of table.

## QA

- Go to the logs page.
- Scroll to the bottom of the page.
- You should see pagination buttons.
- Navigate forward.
- Click the back to start button and it should take you to the first page.

## Details

https://warthogs.atlassian.net/browse/WD-5780
